ack.el

```diff
@@ -314,8 +315,8 @@ This gets tacked on the end of the generated expressions.")
   (interactive)
   (delete-minibuffer-contents)
   (let ((ack (or (car (split-string ack-command nil t)) "ack")))
-    (if (equal ack "ag")
-        (skeleton-insert `(nil ,ack " -ig '" _ "'"))
+    (cond ((equal ack "ag") (skeleton-insert `(nil ,ack " -ig '" _ "'")))
+	  ((equal ack "rg") (skeleton-insert `(nil ,ack " --color always --files --iglob '*" _ "*'")))
```

Comment:
Maybe you could break this very long line?

Reply:
ok, hope it looks better now